### PR TITLE
Integration/riak admin handoff team

### PR DESCRIPTION
**Pull request by andrewjstone**
Should be merged together with corresponding riak and riak_ee PRs for 2.0.4 release.

### CODE REVIEW

File: priv/riak_core.schema [hunk 1]
```diff
@@ -95,6 +95,24 @@
   hidden
 ]}.
 
+%% @doc Disables outbound handoff transfers for this node. If you
+%% turn this setting on at runtime with riak-admin, it will kill any
+%% outbound handoffs currently running.
+{mapping, "handoff.disable_outbound", "riak_core.disable_outbound_handoff", [
```
Comment (seancribbs) [inline priv/riak_core.schema]: Pretty sure @angrycub will comment on this, but in other areas we tended not to have config keys that include the state of the thing in the name, especially when the type is a flag.  How about "handoff.outbound" -> on/off? It's easy to invert the flag, for example:

``` erlang
{mapping, "handoff.outbound", "riak_core.disable_outbound_handoff", [
  {datatype, {enum, [{on, false}, {off, true}]}},
  {default, on},
  hidden
]}.
```

Comment (angrycub) [inline priv/riak_core.schema]: Agreed.  Double negatives are a very sad thing.

Comment (andrewjstone) [inline priv/riak_core.schema]: I agree as well. @nickelization I think we should use @seancribbs idea.

**Comment by andrewjstone** [comment]
:+1:  06a20ad

**Comment by andrewjstone** [comment]
@borshop merge

**Pull request merged**